src/libraries/System.Runtime/tests/System.Diagnostics.Debug.Tests/DebugTestsNoListeners.Interpolation.cs

```diff
@@ -43,6 +43,7 @@ public void Asserts_Interpolation_Syntax()
         }
 
         [Fact]
+        [SkipOnPlatform(TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst, "Trace and debug listeners were processed using ILStrip.")]
```

Comment:
@akoeplinger Please take a look at disabled tests. I didn't find a way to refactor them, but it looks like the ILStrip removes something in trace and debug listeners. If it is a bug, we can report an issue and use `ActiveIssue` annotation.

Reply:
Yeah this looks like a bug. Instead of disabling the tests we could also disable ILStrip for this assembly (e.g set ShouldILStrip=false in the .csproj and then check for ShouldILStrip == '' in AppleBuild.props